plugins/available/nvm.plugin.bash

```diff
@@ -10,9 +10,9 @@ about-plugin 'node version manager configuration'
 
 export NVM_DIR="${NVM_DIR:-$HOME/.nvm}"
 # This loads nvm
-if _bash_it_homebrew_check && [[ -s "${BASH_IT_HOMEBREW_PREFIX}/nvm.sh" ]]
+if _bash_it_homebrew_check && [[ -s "$(brew --prefix nvm)/nvm.sh" ]]
 then
-  source "${BASH_IT_HOMEBREW_PREFIX}/nvm.sh"
+  source "$(brew --prefix nvm)/nvm.sh"
```

Comment:
When we were using a variable that almost-certainly existed with a valid value, the reference was fine.

BUT using the result of a sub-shell that may not return a valid value seems more risky.

ie. When brew is not managing the nvm install, you get a msg on stderr:
```
Error: No available formula with the name "nvm".
```
But the test continues, essentially checking for "/nvm.sh".

I think we need to isolate the fetching of the prefix call and check it if it's valid before using it.

Also, invoking `brew` twice seems a bit heavy.

It also _feels_ like it runs the risk of triggering brew's random update check, although I haven't researched if it can be triggered during a `--prefix` check ...

Reply:
This is only reverting to the old approach. I think that we should probably merge this fix, and then improve BASH_IT_HOMEBREW_PREFIX as a further improvement

Reply:
@davidpfarrell I've attempted a slightly different approach. Can you take a look please?

Reply:
Also, I've verified that `brew --prefix ...` is actually quite a lightweight operation and won't trigger auto-updates: https://github.com/Homebrew/brew/blob/8421b702c5bdc0861b4c794af342d8705e6bc622/Library/Homebrew/brew.sh#L99-L101